README.md

```diff
@@ -129,7 +129,44 @@ This is Day to Day Work Linux Cheatsheet for Software Engineers.
 |Undo| u
 |Redo| ctrl + r
 
+## Bash Commands
+| Name   |   Command |
+|------------ | -------------|
+|Show System| uname -a
+|Show Mounted File System| mount
+|Show System Date| date
+|Show Uptime| uptime
+|Show username| whoami
+|Show Manual Command| man `<command>`
+|Show History Command| history
+
+## Bash Shortcuts
+| Name   |   Command |
+|------------ | -------------|
+|Stop Current Command| CTRL + c
+|Sleep Program| CTRL + z <correct>
+|Search History| CTRL + r <correct>
+|Repeat Last Command| !! <correct>
+|Run Last Command Start with Given character| ! <character>
```

Comment:
OK, I got it, so these should be as below:

|Sleep the current running process | CTRL + z
|Search commands from history | CTRL + r
|Repeat the latest command | !!
|Run the most recent command that matches with starting character(s) of string | !`<string>`

Reply:
Ok